src/getViteConfigPath.test.ts

```diff
@@ -0,0 +1,30 @@
+import { describe, expect, it } from "vitest";
+import path from "path";
+import { getViteConfigPath } from "./getViteConfigPath";
+
+describe("getViteConfigPath", () => {
+  const mockWorkingDirectory = path.resolve(
+    __dirname,
+    "..",
+    "test",
+    "mockConfig"
+  );
+
+  it("resolves with a full path if a file at the provided path exists", async (): Promise<void> => {
+    await expect(
+      getViteConfigPath(mockWorkingDirectory, "vitest.config.all.js")
+    ).resolves.toMatch(/test\/mockConfig\/vitest\.config\.all\.js/i);
+  });
+
+  it("resolves with a full path if no path is provided but a file with a default name exists", async (): Promise<void> => {
+    await expect(getViteConfigPath(mockWorkingDirectory, "")).resolves.toMatch(
+      /test\/mockConfig\/vitest\.config\.js/i
+    );
+  });
+
+  it("rejects if config file can not be found", async (): Promise<void> => {
+    await expect(
+      getViteConfigPath(mockWorkingDirectory, "doesNotExist")
+    ).rejects.toThrow(/unable to find config file/i);
```

Comment:
As this test will execute `core.setFailed()` in the actual source-file, this will always print an error to the summary (as you can see [here](https://github.com/davelosert/vitest-coverage-report-action/actions/runs/4446628502?pr=168).

Would you mind mocking the `core.setFailed()` function for this particular test and make it a noop?

Reply:
Of course, no problem at all 👍